app/controllers/home_controller.rb

```diff
@@ -0,0 +1,4 @@
+class HomeController < ApplicationController
+  def index
+  end
+end
```

Comment:
i think u don't need this file (it was generated by the home controller thing when we didn't have a landing page and used home#index)